# HG changeset patch
# User Rik <rik@octave.org>
# Date 1350421437 25200
#      Tue Oct 16 14:03:57 2012 -0700
# Node ID 7d76290fb29b3e55059b0e45946e78e08a571387
# Parent  95d7475a0a8962d32fca27332b80694e53a18078
unimplemented.m: Remove gco and fminsearch from list of missing functions.

* unimplemented.m: Remove gco and fminsearch from list of missing functions.

diff --git a/scripts/help/unimplemented.m b/scripts/help/unimplemented.m
--- a/scripts/help/unimplemented.m
+++ b/scripts/help/unimplemented.m
@@ -174,23 +174,21 @@ function list = missing_functions ()
   "expint",
   "export2wsdlg",
   "figurepalette",
   "filebrowser",
   "fill3",
   "fitsinfo",
   "fitsread",
   "flow",
-  "fminsearch",
   "frame2im",
   "freqspace",
   "funm",
   "gallery",
   "gammaincinv",
-  "gco",
   "getframe",
   "getpixelposition",
   "grabcode",
   "graymon",
   "gsvd",
   "guidata",
   "guide",
   "guihandles",
